Travel_fellowships.md

```diff
@@ -59,7 +59,7 @@ This section is important for the evaluation of their application.
 
 ### Justification for funding
 
-The OBF Event Fellowships are available for the awardees to cover conference registration fees and other participation-related costs of up to USD 1,000. 
+The OBF Event Fellowships are available for the awardees to cover conference registration fees and other participation-related costs of up to USD $1,000. 
```

Comment:
```suggestion
The OBF Event Fellowships are available for the awardees to cover conference registration fees and other participation-related costs of up to USD 1,000. 
```
The dollar sign is ambiguous (there are a number of countries that use "dollar" as the name of their currency), and redundant with the actually very specific USD (US Dollar). (I.e., USD $1 literally reads _1 US dollar dollar_)